sdk/inc/azure/core/az_mqtt5_connection_config.h

```diff
@@ -14,40 +14,41 @@
 
 #include <azure/core/az_config.h>
 
-#ifndef AZ_MQTT5_CONNECTION_DEFAULT_MIN_RETRY_DELAY_MSEC
+#ifndef AZ_MQTT5_CONNECTION_MIN_RETRY_DELAY_MSEC
 /**
- * @brief The default minimum retry delay in milliseconds.
+ * @brief The minimum delay in milliseconds between retry attempts to connect to the MQTT 5 broker.
  */
-#define AZ_MQTT5_CONNECTION_DEFAULT_MIN_RETRY_DELAY_MSEC (1000)
-#endif // AZ_MQTT5_CONNECTION_DEFAULT_MIN_RETRY_DELAY_MSEC
+#define AZ_MQTT5_CONNECTION_MIN_RETRY_DELAY_MSEC (1000)
```

Comment:
If the user wants to define another value for this, where would they be expected to do it, and have you tried it? When I tried to add an ifndef for a different work item, defining another value in the sample didn't compile properly.

Reply:
The user can define this in their own configuration file